rclpy/docs/source/conf.py

```diff
@@ -194,9 +194,9 @@
 
 # -- Extension configuration -------------------------------------------------
 
-autoclass_content = 'both'
+#autoclass_content = 'both'
```

Comment:
Why was this setting commented out? Seems like we could leave it how it was.

Reply:
sorry, it seems that I forgot to revert it back while experimenting!